### PR TITLE
Fix scaffolder backend bug with visibility for github org

### PR DESCRIPTION
**Pull request by ebarriosjr**
Signed-off-by: ebarrios <esteban.barrios@trivago.com>

## Hey, I just made a Pull Request!

This fixed a bug on the repo creation for create in org. Based on #5007 

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: .changeset/blue-sheep-act.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-backend': patch
+---
+
+Update visibility for create in repo function
```
Comment (jhaals) [inline .changeset/blue-sheep-act.md:5]:
```suggestion
Fixes bug in the `github:publish` action causing repositories to be set as private even if the visibility is set to internal
```

**Comment by jhaals** [comment]
Thanks for fixing this, small nitpick in the changeset but otherwise 👍

**Review by jhaals — APPROVED** [review]

**Review by freben — APPROVED** [review]

**Comment by freben** [comment]
I guess the "public per default" stance is more risky than the inverse. But if I understand things right, this was required right now

**Comment by benjdlambert** [comment]
I think that the default argument that is passed to the action is still `repoVisibility === private` :)

**Pull request merged**